config/initializers/persistence.rb

```diff
@@ -0,0 +1,30 @@
+#==============================================================================
+# Copyright (C) 2020-present Alces Flight Ltd.
+#
+# This file is part of FlightSchedulerController.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# FlightSchedulerController is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with FlightSchedulerController. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on FlightSchedulerController, please visit:
+# https://github.com/openflighthpc/flight-scheduler-controller
+#==============================================================================
+
+FlightScheduler.app.job_registry.load
+FlightScheduler.app.allocations.load
+FlightScheduler.app.init_periodic_processors
```

Comment:
We've lost the guard against not running this during tests.  When controller specs are ran, the registries are loaded with whatever state has been persisted potentially making them non-deterministic.  The periodic processors could also result in unexpected behaviour in tests.  Please add the guard back.

Reply:
No need for the guard as `config/initializers/persistence.rb` as required in `config.ru`. Requiring `config.ru` breaks rspec as the rack DSL hasn't been loaded. This means there is a low chance of it accidentally being required in the future.

Reply:
I've added the guard back.  It doesn't change the behaviour but it does document that these actions should not happen under the test environment.  Explicitly documenting this next to the code is preferable.